scripts/ndk-make.sh

```diff
@@ -66,6 +71,23 @@ export CARGO_TARGET_X86_64_LINUX_ANDROID_LINKER="$TOOLCHAIN/bin/x86_64-linux-and
 
 export RUSTUP_TOOLCHAIN=$(cat "$(dirname "$0")/rust-toolchain")
 
+if test "$1" = "--debug"; then
+  echo Fast, partial, slow debug build. DO NOT UPLOAD THE APK ANYWHERE.
```

Comment:
```suggestion
  echo Quick debug build that will produce a slower app. DO NOT UPLOAD THE APK ANYWHERE.
```
"Partial" is not true here anymore